### PR TITLE
[portal-test-integration] LPS-102243

### PR DESCRIPTION
**Pull request by dantewang**
@tinatian All classes in portal-test-integration are deprecated. I cleaned up some of the gradle dependencies and runtime classpaths, but kept building and deploying of portal-test-integration untouched. I didn't touch the hidden dependency for testCompile or testIntegrationCompile either.

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test

File: portal-test/src/com/liferay/portal/kernel/service/test/ServiceTestUtil.java [hunk 1]
```diff
@@ -86,232 +32,7 @@
...
-
-		calendar.set(Calendar.MONTH, month);
-		calendar.set(Calendar.DATE, day);
-		calendar.set(Calendar.YEAR, year);
-
-		return calendar.getTime();
-	}
-
-	public static void setUser(User user) throws Exception {
+	public static void setUser(User user) {
```
Comment (tinatian) [inline portal-test/src/com/liferay/portal/kernel/service/test/ServiceTestUtil.java:35]: @dantewang @ChiZelin Please merge this class into `com.liferay.portal.kernel.test.util.UserTestUtil`. There is only one method left and it is related to user.

Tina.

File: portal-test/src/com/liferay/portal/kernel/service/test/SynchronousInvocationHandler.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
  * details.
  */
```
Comment (tinatian) [inline portal-test/src/com/liferay/portal/kernel/service/test/SynchronousInvocationHandler.java:14]: @dantewang @ChiZelin 
Please move this class to `com.liferay.portal.kernel.test`.

Tina

File: portal-impl/test/unit/com/liferay/portal/dao/sql/transformer/TestDB.java [hunk 3]
```diff
@@ -12,9 +12,10 @@
  * details.
  */
 
-package com.liferay.portal.dao.db;
+package com.liferay.portal.dao.sql.transformer;
```
Comment (tinatian) [inline portal-impl/test/unit/com/liferay/portal/dao/sql/transformer/TestDB.java:15]: @dantewang @ChiZelin 
I remembered that we have an issue about ee test db classes, if we move the base classes to portal-impl unit test package, modules can not see it. So we are clear now?

Tina.

**Comment by tinatian** [comment]
@dantewang @ChiZelin Please see comments and resend.

Thanks.
Tina

**Pull request closed without merging**